src/Sylius/Bundle/AttributeBundle/Controller/AttributeController.php

```diff
@@ -48,6 +48,8 @@ public function getAttributeTypesAction(Request $request, $template)
      */
     public function renderAttributesAction(Request $request)
     {
+        $template = $request->attributes->get('template', 'SyliusAttributeBundle::attributeChoice.html.twig');
```

Comment:
Should this be `->get('_sylius[template]', 'SyliusAttributeBundle::attributeChoice.html.twig');` to follow the standard convention?